docs(readme): added a contributing section with commit message (#132)
coveo/platform-client: README.md

    notifyOnError('It failed!'),
).field.delete('unwanted-field-id');
```
## Contributing

### Guidelines

> -   Make sure your changes (in TypesScript files) are fully tested.

Reviewer: ```suggestion
-   Make sure your changes are fully tested (when applicable).
```

    notifyOnError('It failed!'),
).field.delete('unwanted-field-id');
```
## Contributing

### Guidelines

-   Make sure your changes are fully tested (when applicable).
-   We tend to avoid comments in our code base, we strongly prefer good naming and code structure.

> -   Avoid pushing similar changes in different commits if it's within the same feature, because we want the changelogs to be clear and simple. To avoid that, there are at least 2 options : 
> 1. Squash the commits into one when merging (you need to edit the final commit message though)
> 1. amend the previous commit when making related changes (`git commit --amend --no-edit`)

Reviewer: ```suggestion
-   Avoid pushing similar changes in different commits if it's within the same feature, because we want the changelogs to be clear and simple. To avoid that, there are at least 2 options: 
  1. Squash the commits into one when merging (you need to edit the final commit message though)
  1. amend the previous commit when making related changes (`git commit --amend --no-edit`)
```
